public/components/integrations/components/add_integration_flyout.tsx

```diff
@@ -131,10 +131,16 @@ export function AddIntegrationFlyout(props: IntegrationFlyoutProps) {
   const fetchDataSourceMappings = async (
     targetDataSource: string
   ): Promise<{ [key: string]: { properties: any } } | null> => {
-    return fetch(`/api/console/proxy?path=${targetDataSource}/_mapping&method=GET`, {
-      method: 'POST',
-      headers: [['osd-xsrf', 'true']],
-    })
+    return fetch(
+      http.basePath.prepend(`/api/console/proxy?path=${targetDataSource}/_mapping&method=GET`),
```

Comment:
why is this going through console (devtools) 's api?

also i think if you use `http.get('/api/console')` or `http.get('./api/console')` it works without manually adding base path